pa11ycrawler/templates/index.html

```diff
@@ -0,0 +1,44 @@
+<%inherit file="base.html" />
+<%namespace name="summary" file="summary.html"/>
+<div class="col-sm-10 col-sm-offset-1">
+<h1>Accessibility Audit Index</h1>
+</div>
+${summary.make_summary(
+    count['error'],
+    count['warning'],
+    count['notice'],
+    count['total']
+)}
+
+<br />
+<div class="col-sm-10 col-sm-offset-1" style="margin-bottom: 30px;">
+<h2 class="hd-4">Pages Checked</h2>
+<table data-classes='table table-no-bordered'
+    data-toggle="table"
+    data-search="true">
+    <thead>
+        <tr>
+            <th data-field='URL' data-sortable="true">URL</th>
+            <th class='active' data-field='Total' data-sortable="true">Total</th>
+            <th class='danger' data-field='Errors' data-sortable="true">Errors</th>
+            <th class='warning' data-field='Warnings' data-sortable="true">Warnings</th>
+            <th class='info' data-field='Notices' data-sortable="true">Notices</th>
+        </tr>
+    </thead>
+    <tbody>
+        % for url,result in pageResults.iteritems():
+            ${makerow(result, url)}
+        % endfor
+    </tbody>
+</table>
+</div>
+
+<%def name="makerow(result, url)">
+    <tr>
+        <td>${url} (<a href="${result['filename'] + '.html'}">See results<span class='sr-only'>&nbsp;for ${url}</span></a>)</td>
```

Comment:
Yea, it breaks in the middle of words (e.g. urls) right now.   I've gotten some feedback from Mark to change this column to display page title as a link to the url.  That should help with the long url and readability issue.